README.md

```diff
@@ -6,58 +6,57 @@ You will find here the default rules for the rooms part of the PSA Community.
 
 *We are not lawyers.*  
 *We enforce the spirit of the rules, not the letter.*  
 *Citing technicalities or imprecisions in the definitions of rules won’t save you.* 
 
 **Avoid drifting too much off-topic or move to off-topic rooms like #psa-offtopic or #bnonymity** 
-- Keep it legal (talks about Sci-Hub and/or LibGen, Z-lib, etc. are allowed)
+
+- Keep it legal
 - English only
 - Be respectful to each other
 - Avoid FUD and/or disinformation
 - Avoid gatekeeping and try to remain welcoming to new users
-- No hate speech (no racism, no homophobia, no transphobia)
+- No hate speech
 - If asked, please respect someone's pronouns)
 - No spam
-- No doxxing
+- No doxxing (unless specificaly requested by the person, which you should then take to a DM).
 - No trolling (this doesn't mean sarcasm is forbidden)
 - No NSFW content (no Porn, no Gore, no Hentai)
 - No upload of any non-media files (binaries, executables, archives, etc.)
 - No voice messages (these will be auto-deleted by the bots)
+- Limit political discussions to room topics related issues only.
 
 #### Serious Offenses:<a name="serious"></a>
-Some exceptions can apply. Violations will be handled at the discretion of the acting moderator.
-- **Limit political discussions to privacy-related topics only**
-- Zero tolerance for discussion of how to commit illicit acts.  
+Some exceptions can apply. Violations will be handled at the discretion of the acting moderator or admin.
+- Zero tolerance for discussion inciting illicit, unethical, hate acts. It is however permitted to discuss these acts as part of a discussion but without actively engaging in inciting or committing those.
   - *Talks about torrenting anonymously are allowed unless the purpose is blatantly illegal.*
-- Do not dox others
-- No suspicious links or uploading of non-image binary files
-- Be respectful
-- **Any** discussion about AnonyPla/Lena's whereabouts is tolerated, to an extent, only in the ```#Bnonymity:matrix.org``` room.
+- Any CSAM type offense will result in a permanent ban without a possibility of appeal.
+  
+#### Special Rules for Anonymity room:<a name="anon"></a>
+- **Any** discussion about AnonyPla/Lena's whereabouts is not tolerated but will be tolerated, to an extent, in the ```#Bnonymity:matrix.org``` room.
+-  Talks about Sci-Hub and/or LibGen, Z-lib, etc. are allowed in this room.
 
 #### Rules for Modern Cryptography (```#moderncrypto:gnuradio.org```):<a name="moderncrypto"></a>
 See <https://anonymousplanet.org/moderncrypto-rules.html>
 
-#### Rules for OS Security (```#os-security:artemislena.eu```):<a name="ossecurity"></a>
-See <https://artemislena.eu/coc.html>
-
 #### Rules for Bnonymity (```#bnonymity:matrix.org```):<a name="bnonymity"></a>
 - Keep it legal (seriously)
 - English only (no Russian, no Chinese, English only)
-- Be "somewhat" respectful to each other
-- No hate speech (No racism, no homophobia, no transphobia...)
+- Be "somewhat" respectful to each other but this is "optional"
```

Comment:
```suggestion
- Be kind, don't be a problem
```

Reply:
> I disagree, shitposting and some level of trolling and slurs have to be accepted in that room as long as those are not hate speech. Like saying "F*** You" is ok to me in Bnon ...

I was more referring to "f-ck this <*entire group of humans*>" which goes hand-in-hand, e.g. with xenophobia, transphobia, etc.

I don't care about a f-bomb here and there, but if it's directed at entire group, it's not acceptable imo. The groups most people are going to come into contact with hate speech are transgender men and women, so I used that as an example of the most common types of abuse one might encounter in modern chat circles. We need to be as inclusive as possible.